pkg/disttask/framework/taskexecutor/manager.go

```diff
@@ -132,13 +132,15 @@ func (m *Manager) initMeta() (err error) {
 	return err
 }
 
+// InitMeta initializes the meta of the Manager.
+// not a must-success step before start manager, manager will try to init meta periodically.
+func (m *Manager) InitMeta() error {
```

Comment:
Just rename initMeta to InitMeta？

Reply:
it's named `Init` in the begin, but it's misleading to Start even Init failed, so renamed to `InitMeta`

will leave it for now